### DISCUSSION TITLE
ValueError: Header hash b'...' not in the blockchain

### DISCUSSION THREAD

**Discussion by ast0815** [Chia-Network/chia-blockchain#8921] [Support]
After a power loss of my computer, my full node does no longer keep up with the main chain. At the end of the log it says:

```
ValueError: Header hash b'\xa2\xac\x05J\xe6\xa3E\xab\xa4\x8fH}\xac\xfa\x9f\x9a]\x91\x0f\x10bn\xa3\x1e\xcb\xec2\xd69\x06L\xfd' not in the blockchain
```

The full node process is still running at 100% CPU, but the status just switches between "Not Synced" and "Syncing 1037240/<an increasing number goes here>". So it seems like it keeps track of the fact that the chain gets longer, but does no longer update the local one.

Is there a way for me to fix this? Does the 100% CPU load mean that it is trying to find the error in the local chain and will get back to working if left alone for long enough? Or do I have to delete the database and let it sync from 0 again?

I'd like to avoid the latter, so any help is appreciated.

**Comment by ast0815** [comment]
Just to close this: I ended up deleting the DB and syncing from scratch.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
